e2e/config/config.go

```diff
@@ -237,6 +237,31 @@ func (c configTests) configGlobal(t *testing.T) {
 			exit:           0,
 			resultOp:       e2e.ExpectOutput(e2e.ExactMatch, "1"),
 		},
+		{
+			name:           "AllowUserNsNo",
+			argv:           []string{"--userns", c.env.ImagePath, "true"},
+			profile:        e2e.UserProfile,
+			directive:      "allow user ns",
+			directiveValue: "no",
+			exit:           1,
+		},
+		{
+			name:           "AllowUserNsNoFakeroot",
+			argv:           []string{"--fakeroot", c.env.ImagePath, "true"},
```

Comment:
Instead of `--fakeroot` as a flag, we use `e2e.FakerootProfile` in the e2e tests, by convention.